app/models/spree/payment_decorator.rb

```diff
@@ -73,6 +73,15 @@ def refund!(refund_amount=nil)
       end
     end
 
+    # Import from future Spree
+    def build_source
+      return if source_attributes.nil?
+      if payment_method and payment_method.payment_source_class
+        self.source = payment_method.payment_source_class.new(source_attributes)
+        self.source.payment_method_id = payment_method.id
```

Comment:
Hey @stveep, I am a bit worried about this line. I don't think it is going to work for us to link instances `Spree::CreditCard` to a single `Spree::PaymentMethod` - this will imply that each card can only be used by a single shop, and we want cards to be used by multiple shops.... Was there a particular reason that we need this association as far as you're aware? I realise this method is coming from Spree, so they must have a reason but I suspect it is more for convenience than actual necessity, because the payment itself stores the payment method, so I don't see why the card needs to store it as well...

Reply:
Bump @stveep